radio/src/gui/horus/view_channels.cpp

```diff
@@ -137,3 +182,38 @@ bool menuChannels2(evt_t event)
   
   return menuChannelsMonitor(event, 1);
 }
+
+
+bool menuMixers1(evt_t event)
+{
+  MENU("Mixer monitor 1-16", MONITOR_ICONS, menuTabMonitors, e_Mixers1, 0, { 0 });

```

Comment:
oh. And I did this by hand for all files in my pull requests. I feel stupid now.